src/main/scala/mesosphere/marathon/io/IO.scala

```diff
@@ -85,6 +85,15 @@ object IO {
     new String(out.toByteArray, "UTF-8")
   }
 
+  def withResource[T](path: String)(fn: InputStream => T): Option[T] = {
```

Comment:
I compute the normalized path and make sure, only /META-INF/resources/webjars/* is accessible.